library/src/scala/compiletime/ops/package.scala

```diff
@@ -6,39 +6,33 @@ package object ops {
   @infix type ==[X <: AnyVal, Y <: AnyVal] <: Boolean
   @infix type !=[X <: AnyVal, Y <: AnyVal] <: Boolean
 
-  @infix type +[X <: Int | String, Y <: Int | String] = (X, Y) match {
```

Comment:
I actually like this example a lot. It's a shame to delete it. I propose to change it to a test code (or provide it as example in a document).

Reply:
Good point -- I've done both in 2cab5919559cca0836d517b3fc74475d3336fe4c, and generally documented the feature.